app/src/ccc36c3/res/values-pl/strings.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Grafik 36C3</string>
+    <string name="app_hardcoded_subtitle">27-30 grudnia 2019, Lipsk, Leipziger Messe</string>
```

Comment:
Android Studio (Lint) suggests the following here:

> Replace "-" with an "en dash" character (–, `&#8211;`) ?

![dash](https://user-images.githubusercontent.com/144518/145778377-c8e2defd-8b81-479d-acad-c2c2cc34a0d0.png)

Please apply the suggestion (ALT + RETURN) here and in the `values-pl/strings.xml` file. Please notet that Lint only highlights such cases in files of the currently selected build flavor.

![image](https://user-images.githubusercontent.com/144518/145779327-faa338d2-e227-441f-aecd-1905a50911a9.png)

Reply:
`-` replaced with `–` (and I agree that this is a better character). Bear in mind that this was taken from the English source, so probably a correction would be also needed right there, but this goes slightly outside of the Polish translation scope, so I am not touching the `values` folder.

Reply:
Plus I also changed the location of the `Logo` text:
```
<string name="about_logo_content_description">
        36C3: Resource exhaustion Logo
</string>
```
with
```
<string name="about_logo_content_description">
        Logo 36C3: Resource exhaustion
</string>
```
as I overlooked that at the very beginning (in Polish the text `logo` goes before what it reflects).

@johnjohndoe, if you are happy with the changes you can resolve this conversation, as I believe you are the right person to do so, not me.